README.md

```diff
@@ -64,6 +64,7 @@ Copy `.env.example` to `.env` and customize it for your environment:
 |DALLE3_DEPLOYMENTS|``|Comma-separated list of deployments that support DALL-E 3 API. Example: `dall-e-3,dalle3,dall-e`|
 |GPT4_VISION_DEPLOYMENTS|``|Comma-separated list of deployments that support GPT-4V API. Example: `gpt-4-vision-preview,gpt-4-vision`|
 |GPT4_VISION_MAX_TOKENS|1024|Default value of `max_tokens` parameter for GPT-4V when it wasn't provided in the request|
+|GPT4O_MAX_TOKENS|4096|Default value of `max_tokens` parameter for GPT-4o when it wasn't provided in the request|
```

Comment:
We use to provide default max_tokens value in one of the cases:
1. The default max_tokens of the model itself is set to be very low (it was the case for GPT4-V, its default was only 16 tokens)
2. The model does allow missing max_tokens argument, so we have to provide one with every request.

Does GPT4-o fall into one of the cases?

Also, keep in mind that the need for max_tokens defaults has lessen significantly since DIAL Core supported `defaults` for chat completion request overrides.

Reply:
Fixed, removed